connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -68,7 +65,6 @@
 
 @RunWith(PowerMockRunner.class)
 public class WorkerSourceTaskTest extends ThreadedTest {
```

Comment:
We could probably add something which provides the same capability for an executor service though. That would be better than sprinkling `future.get()` calls all over the place.